internal/metamorph/processor.go

```diff
@@ -582,69 +583,57 @@ func (p *Processor) StartRequestingSeenOnNetworkTxs() {
 	}()
 }
 
+// Periodically read transactions with status lower then SEEN_ON_NETWORK from database and announce them again
```

Comment:
Change comment to 

`// StartProcessExpiredTransactions periodically reads transactions with status lower then SEEN_ON_NETWORK from database and announce them again`